components/brave_shields/resources/panel/components/main-panel/index.tsx

```diff
@@ -100,15 +110,11 @@ function MainPanel () {
       </S.SiteTitleBox>
       <S.CountBox>
         <S.BlockNote>
-          {braveShieldsBlockedNote.beforeTag}
-          <a href="#" onClick={handleLearnMoreClick}>{braveShieldsBlockedNote.duringTag}</a>
-          {braveShieldsBlockedNote.afterTag}
+          {braveShieldsNote.beforeTag}
+          <a href="#" onClick={handleLearnMoreClick}>{braveShieldsNote.duringTag}</a>
```

Comment:
Sidenote: Perhaps this shouldn't be an anchor element since it's not a link. For these we should use the text version of button. Can be follow-up if we don't have that version done yet.